packaging/windows/internal/cuda_install.bat

```diff
@@ -222,9 +224,9 @@ set "NVTOOLSEXT_PATH=%ProgramFiles%\NVIDIA Corporation\NvToolsExt\bin\x64"
 if not exist "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v%CUDA_VERSION_STR%\bin\nvcc.exe" (
```

Comment:
Shall we move this section to line 195

Reply:
Sounds like a good idea. Let me do it as part of separate PR